Specs/Data/Models/GltfLoader/BoxTextured/glTF/BoxTextured-WithCredits.gltf

```diff
@@ -0,0 +1,182 @@
+{
```

Comment:
In `Specs/Data/Models/GltfLoader` we try to keep the test models separate and self contained. Could you move this model to its own folder in `Specs/Data/Models/GltfLoader`? It would have a single `glTF` subfolder.

Any maybe modify https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/Box instead of the textured box since we don't need the textures.


Also I would suggest renaming to `BoxWithCopyright` since "copyright" is the glTF specific term.